.changeset/clever-planets-crash.md

```diff
@@ -0,0 +1,5 @@
+---
+"apollo-client-devtools": minor
+---
+
+Show a banner with a prompt to restart the page and the devtools instead of crashing the page when the extension can't communicate with the content scripts .
```

Comment:
```suggestion
Show a banner with a prompt to restart the page and the devtools instead of crashing the page when the extension can't communicate with the content scripts.
```